shardingsphere-sql-parser/shardingsphere-sql-parser-statement/src/main/java/org/apache/shardingsphere/sql/parser/sql/common/constant/TransactionIsolationLevel.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.shardingsphere.sql.parser.sql.common.constant;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+/**
+ * Transaction isolation level enum.
+ */
+@RequiredArgsConstructor
+@Getter
+public enum TransactionIsolationLevel {
+
+    NONE("NONE"),
+    REPEATABLE_READ("REPEATABLE_READ"),
+    READ_UNCOMMITTED("READ_UNCOMMITTED"),
+    SERIALIZABLE("SERIALIZABLE"),
+    READ_COMMITTED("READ_COMMITTED");
```

Comment:
Please change the sequence to READ_UNCOMMITTED, READ_COMMITTED, REPEATABLE_READ, SERIALIZABLE